cuML/src/glm/glm.cu

```diff
@@ -180,29 +180,27 @@ void qnFit(float *X, float *y, int N, int D, int C, bool fit_intercept,
            int linesearch_max_iter, int lbfgs_memory, int verbosity, float *w0,
            float *f, int *num_iters, bool X_col_major, int loss_type) {
 
-  // TODO handles will come from the cuml handle
-  cudaStream_t stream = 0;
-  cublasHandle_t cublas_handle;
-  CUBLAS_CHECK(cublasCreate(&cublas_handle));
+  // TODO cuml will be passed in
+  cumlHandle cuml;
+  cuml.setStream(0);
```

Comment:
This is no more needed after @vinaydes's fixes for initializing the stream during the ctor of `cumlHandle`.